hack/update-codegen.sh

```diff
@@ -28,7 +30,7 @@ fi
 root_dir=$(git rev-parse --show-toplevel)
 
 # retrieve the code-generator scripts and bins
-go get -u k8s.io/code-generator/...
+go get k8s.io/code-generator/...
```

Comment:
do we still want the -u to update?

Reply:
This seems non-deterministic to me. I was thinking I would rather deliberately update my generator. WDYT?

Reply:
I think I'd rather us break than have everyone running at slightly different versions.

Maybe we could explicitly do the update on one line, CD into the directory and git checkout a specific tag right below it so the version is explicitly called out?

Reply:
That sounds good. I'll go with `-u` for now as it also has a bunch of deps and isn't using go modules yet.